website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -1,502 +1,120 @@
 ---
-title: Golang Based Operator Quickstart
-linkTitle: Quickstart
-weight: 10
+title: Golang Based Operator QuickStart
+linkTitle: QuickStart
+weight: 20
 ---
 
-**NOTE:** For the SDK versions prior to `v0.19.0` please consult the [legacy docs][legacy_quickstart_doc] for the [legacy CLI][legacy_CLI] and project.
-
-This guide walks through an example of building a simple memcached-operator using the operator-sdk CLI tool and controller-runtime library API.
-
 ## Prerequisites
 
 - [go][go_tool] version v1.13+.
 - [docker][docker_tool] version 17.03+.
 - [kubectl][kubectl_tool] version v1.11.3+.
-- [kustomize][kustomize_tool] v3.1.0+
+- [operator-sdk][operator_install] v.0.19+
```

Comment:
I don't think its necessary to list our own project as a prerequisite. I think that should be obvious.